front/app/components/admin/ActionForm/FlowVisualization/Blocks/VerificationModal.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+
+import {
+  Box,
+  Title,
+  Text,
+  Icon,
+  fontSizes,
+  colors,
+} from '@citizenlab/cl2-component-library';
+
+import useVerificationMethod from 'api/verification_methods/useVerificationMethod';
+
+import useLocalize from 'hooks/useLocalize';
+
+import Modal from 'components/UI/Modal';
+
+import { FormattedMessage } from 'utils/cl-intl';
+
+import messages from './messages';
+import { getVerifiedDataList } from './utils';
+
+interface Props {
+  opened: boolean;
+  onClose: () => void;
+}
+
+const VerificationModal = ({ opened, onClose }: Props) => {
+  const { data: verificationMethod } = useVerificationMethod();
+  const localize = useLocalize();
+
+  const verificationMethodMetadata =
+    verificationMethod?.data.attributes.action_metadata;
+
+  if (!verificationMethodMetadata) return null;
+
+  const verifiedDataList = getVerifiedDataList(
+    verificationMethodMetadata,
+    localize
+  );
+
+  return (
```

Comment:
This is this modal that opens if you click the `VerificationBlock`:
![Screenshot 2024-08-27 at 15 56 32](https://github.com/user-attachments/assets/c324435c-a5ed-4d93-905c-d20d7e3462e8)